ui/app/AppLayouts/Chat/panels/communities/ChatPermissionQualificationPanel.qml

```diff
@@ -0,0 +1,122 @@
+                    Row {
+                        spacing: d.panelRowSpacing
+                        Row {
+                            spacing: d.panelRowSpacing
+                            Repeater {
+                                model: HoldingsSelectionModel {
+                                    sourceModel: holdingsListModel
+                                    assetsModel: root.assetsModel
+                                    collectiblesModel: root.collectiblesModel
+                                }
+                                StatusListItemTag {
+                                    height: 20
+                                    enabled: false
+                                    leftPadding: 2
+                                    title: model.text
+                                    asset.name: model.imageSource
+                                    asset.isImage: true
+                                    asset.bgColor: "transparent"
+                                    asset.height: 16
+                                    asset.width: asset.height
+                                    asset.bgWidth: asset.height
+                                    asset.bgHeight: asset.height
+                                    asset.color: asset.isImage ? "transparent" : titleText.color
+                                    closeButtonVisible: false
+                                    titleText.color: model.available ? Theme.palette.primaryColor1 : Theme.palette.dangerColor1
+                                    bgColor: model.available ? Theme.palette.primaryColor2 :Theme.palette.dangerColor2
+                                    titleText.font.pixelSize: 12
+                                }
+                            }
+                        }
+                        StatusBaseText {
+                            height: parent.height
+                            visible: (index !== (repeater.count - 1))
+                            verticalAlignment: Text.AlignVCenter
+                            font.pixelSize: Style.current.primaryTextFontSize
+                            rightPadding: d.panelRowSpacing
+                            color: Style.current.secondaryText
+                            text: qsTr("or")
+                        }
+                    }
+                }
+            }
+        }
+
+        StatusBaseText {
+            Layout.preferredWidth: 30
+            Layout.fillHeight: true
+            font.pixelSize: Style.current.primaryTextFontSize
+            color: Style.current.secondaryText
+            verticalAlignment: Text.AlignVCenter
+            text: qsTr("or... ")
+            MouseArea {
+                anchors.fill: parent
+                cursorShape: Qt.PointingHandCursor
+                onClicked: {
+                    Global.openPopup(communityTokenPermissionsPopup);
+                }
+            }
+        }
+        Item { Layout.fillWidth: true }
+    }
+
+    Component {
```

Comment:
An alternative to not pass all the 3 models is to instantiate this popup outside and from here only raise a signal when it needs to be open.. Just an idea if it makes the component api clearer. Then you can keep only the new property added `holdingsModel` and remove the rest of models.